Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: build run test remove push-tag push-latest login
+
+NAME   := os-viewer
+REPO   := openspending/${NAME}
+TAG    := $(shell git log -1 --pretty=format:"%h")
+IMG    := ${REPO}:${TAG}
+LATEST := ${REPO}:latest
+
+build:
+	docker build -t ${IMG} -t ${LATEST} .
+
+run:
+	docker run ${RUN_ARGS} --name ${NAME} -d ${LATEST}
+
+test:
+	docker ps | grep latest
+	docker exec ${NAME} npm test
```

Comment:
I like that you're running the tests inside docker, but this requires development dependencies to be installed on it (including test DBs), and can be cumbersome when developing (as you either need to rebuild the images or share the directory).

Maybe your thought was that this would only be used on Travis? If so, maybe just changing the name to something like `make ci-test` or similar, so people don't confuse that `make test` is how the test should be run locally.